templates/pastes.html

```diff
@@ -0,0 +1,74 @@
+{% extends 'base.html' %}
+{% block head %}
+  {% load static %}
+{% endblock %}
+{% block content %}
+  <section class="page">
+    <header>
+      <figure>
+        <img src="{% static "/assets/images/gridpaste-logo.svg" %}" alt="logo" />
```

Comment:
IME asset paths are typically relative - that is, not starting with a '/'

They won't be rendered that way - it's a path relative to the static root.